platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -119,10 +121,10 @@
      *                                     objects.
      */
     public ApplicationServiceBean(ApplicationService appService,
-            ConfigurationAdminExt configAdminExt, MBeanServer mBeanServer)
+            ConfigurationAdmin configAdmin, MBeanServer mBeanServer)
             throws ApplicationServiceException {
         this.appService = appService;
-        this.configAdminExt = configAdminExt;
+        this.configAdmin = configAdmin;
         this.mBeanServer = mBeanServer;
```

Comment:
❗️ You're setting this here, and then on line 132 you're calling `ManagementFactory.getPlatformMBeanServer()` and setting the value of the `mBeanServer` _parameter_ passed in. I think line 132 should just be deleted.